docs/one-click-integrations.md

```diff
@@ -10,13 +10,41 @@ To set up the PagerDuty integration:
 
 1.  On Pulse, click **Integrations** and select **PagerDuty**.
 
-    ![PagerDuty integratio](images/pagerduty.png)
+    ![PagerDuty integration](images/pagerduty.png)
 
 1.  Fill in the necessary details:
 
-    -    **PagerDuty subdomain:** Your custom PagerDuty subdomain. For example, `mysubdomain` if you use the URL `https://mysubdomain.pagerduty.com` to access PagerDuty.
-    -    **Pulse system name:** Name of the [system](https://docs.pulse.codacy.com/#before-you-begin) to associate with the incidents reported by this integration.
+    - **PagerDuty subdomain:** Your custom PagerDuty subdomain. For example, `mysubdomain` if you use the URL `https://mysubdomain.pagerduty.com` to access PagerDuty.
+    - **Pulse system name:** Name of the [system](https://docs.pulse.codacy.com/#before-you-begin) to associate with the incidents reported by this integration.
 
 1.  Follow the instructions on the Pulse UI to create a new webhook on PagerDuty using the provided URL. For example:
 
     ![PagerDuty webhook](images/pagerduty-webhook.png)
+
+## GitHub
+
+Pulse integrates directly with GitHub to receive data about deployments and changes.
+
+To set up the GitHub integration:
+
+1.  On Pulse, click **Integrations** and select **GitHub**.
+
+    ![GitHub integration](images/ghi-setup.png)
+
+1.  Click **Install GitHub App**
+
+1.  Follow the instructions on the GitHub UI to install the app in the desired organization.
+
+    ![GitHub webhook](images/ghi-github-install.png)
+
+1.  Wait while Pulse creates the integration and the webhook on GitHub for you.
+
+    ![GitHub webhook](images/ghi-creating.png)
+
+1.  In case of success you should be redirected to a page like:
+
+    ![GitHub webhook](images/ghi-ok.png)
+
+1.  If there is an error as displayed bellow please [contact support](mailto:pulsesupport@codacy.com).
```

Comment:
```suggestion
1.  If there is an error as displayed below please [contact support](mailto:pulsesupport@codacy.com).
```

TIL:
![image](https://user-images.githubusercontent.com/11428771/105383230-cc2e1d00-5c08-11eb-853a-ebaaa65fa648.png)

Reply:
I see this typo a lot, most probably because the spell-checkers don't signal the word since it **does** exist. :slightly_smiling_face:

Reply:
That's exactly what I was thinking, because I'm pretty sure @rtfpessoa has a spell-checker on VSCode